src/main/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -77,8 +83,8 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
            match="*[contains(@class, ' topic/table ')][*[contains(@class, ' topic/title ')]]"
            use="'include'"/>
   
-  <xsl:key name="id" match="*[@id]" use="@id"/>
-  <xsl:key name="topic" match="*[@id][contains(@class, ' topic/topic ')]" use="@id"/>
+  <xsl:key name="elementsById" match="*[@id]" use="@id"/>
```

Comment:
by "elements" I mean "all elements", which is what the original "id" table matched.

But there is no need to be able to find topics and non-topic elements by ID in the same lookup so I think a better name would be "nontopicElementsByID" and have it exclude topics.